.github/workflows/tv-os-build-test.yml

```diff
@@ -0,0 +1,22 @@
+name: Test tvOS build
+on:
+  pull_request:
+    branches: [master]
+jobs:
+  test:
+    runs-on: macos-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          cache: 'yarn'
+          cache-dependency-path: 'TVOSExample/yarn.lock'
+      - name: Install node dependencies
+        run: cd TVOSExample && yarn
+      - name: Install pods
+        run: cd TVOSExample/ios && pod install
+      - name: Build app
+        run: cd TVOSExample && yarn tv-os
```

Comment:
Use `working-directory` rather than changing directory on every step:
https://stackoverflow.com/a/58142276/9999202
Same for other builds